srcpkgs/sioyek/template

```diff
@@ -1,19 +1,20 @@
 # Template file for 'sioyek'
 pkgname=sioyek
-version=1.4.0
+version=1.5.0
 revision=1
 build_style=qmake
 hostmakedepends="qt5-qmake qt5-host-tools
  qt5-opengl qt5-gui qt5-widgets qt5-network"
 makedepends="zlib-devel qt5-3d-devel qt5-gamepad-devel mupdf-devel
- gumbo-parser-devel harfbuzz-devel qt5-opengl mujs-devel"
+ gumbo-parser-devel harfbuzz-devel qt5-opengl mujs-devel freetype-devel
+ libjbig2dec libjpeg-turbo-devel libopenjpeg2-devel"
 depends="mupdf"
 short_desc="PDF viewer designed for reading research papers and technical books"
 maintainer="dkwo <npiazza@disroot.org>"
 license="GPL-3.0-or-later"
 homepage="https://sioyek.info/"
 distfiles="https://github.com/ahrm/sioyek/archive/refs/tags/v${version}.tar.gz"
-checksum=44d49aec28e49bb79c2d0fb7cefd26aecc53b60136bf02dfec9863ac586aacd0
+checksum=2d01c757953c3f2d98428376e642d13b8a9edeba8fee506477e8aa178756f16b
 nocross="fails to configure"
```

Comment:
Could you restrict the arch to i686* and x86_64* instead, even natively it does not build for others

Reply:
what? yes it does, the switch from nocross is absolutely wrong

Reply:
I tried to build natively on aarch64 and it failed.
I think this is true at least also for arm*.
I assumed it would also fail for all others